staging/src/k8s.io/apimachinery/pkg/api/validation/generic.go

```diff
@@ -41,15 +41,15 @@ func NameIsDNSSubdomain(name string, prefix bool) []string {
 	return validation.IsDNS1123Subdomain(name)
 }
 
-// NameIsDNSLabel is a ValidateNameFunc for names that must be a DNS 1123 label.
+// NameIsDNSLabel is a ValidateNameFunc for names that must be a RFC 1123 label.
 func NameIsDNSLabel(name string, prefix bool) []string {
 	if prefix {
 		name = maskTrailingDash(name)
 	}
 	return validation.IsDNS1123Label(name)
 }
 
-// NameIsDNS1035Label is a ValidateNameFunc for names that must be a DNS 952 label.
+// NameIsDNS1035Label is a ValidateNameFunc for names that must be a RFC 1035 label.
```

Comment:
The function IsDNS1035Label is like
```
// IsDNS1035Label tests for a string that conforms to the definition of a label in
// DNS (RFC 1035).
func IsDNS1035Label(value string) []string {
        var errs []string
        if len(value) > DNS1035LabelMaxLength {
                errs = append(errs, MaxLenError(DNS1035LabelMaxLength))
        }
        if !dns1035LabelRegexp.MatchString(value) {
                errs = append(errs, RegexError(dns1035LabelErrMsg, dns1035LabelFmt, "my-name", "abc-123"))
        }
        return errs
}
```
Then it seems fine to change 952 to 1035 at least.
`DNS 1035 label` is not so bad from the above docstring.
`label in DNS (RFC 1035)` would be another option if we want to change here.

Reply:
I've changed the text, thanks for the review